arch/AArch64/AArch64Mapping.c

```diff
@@ -2500,7 +2525,7 @@ void AArch64_set_detail_op_imm(MCInst *MI, unsigned OpNum,
 		if (AArch64_get_detail_op(MI, 0)->type == AARCH64_OP_SME) {
 			AArch64_set_detail_op_sme(MI, OpNum,
 						  AARCH64_SME_MATRIX_SLICE_OFF,
-						  AARCH64LAYOUT_INVALID, 1);
+						  AARCH64LAYOUT_INVALID, (int64_t) 1);
```

Comment:
Alternatively you could use something like `1LL`

Reply:
Is `LL` always equivalent to a 64bit signed integer?

Reply:
Yes, it's a shorthand for `long long`